mm2src/coins/lightning.rs

```diff
@@ -1427,4 +1427,10 @@ impl MmCoin for LightningCoin {
     fn on_disabled(&self) -> Result<(), AbortedError> { AbortableSystem::abort_all(&self.platform.abortable_system) }
 
     fn on_token_deactivated(&self, _ticker: &str) {}
+
+    fn is_available(&self) -> bool { true }
+
+    fn update_is_available(&self, _to: bool) {
+        warn!("`update_is_available` is ineffective for lightning protocol");
+    }
```

Comment:
I might be wrong on this. Please let me know if I should change this behaviour @shamardy

Reply:
You can use
```rust
`warn!("Layer 2 coin {} can't be passive", self.ticker());`
```
since L2/lightning is added to ctx similar to how tokens are added and depends heavily on the onchain coin although it's not similar to other platform coins since fees are not paid by the onchain coin but instead channels APIs/operations depend on it.
https://github.com/KomodoPlatform/atomicDEX-API/blob/a6046fd79af444b7a1f83eb7d388bd8f8c85bcb8/mm2src/coins_activation/src/l2/init_l2.rs#L208
https://github.com/KomodoPlatform/atomicDEX-API/blob/57be79eab446a62496e5a48694005abb6c8386a5/mm2src/coins/lp_coins.rs#L2513-L2515

Reply:
Just an interesting topic I thought to bring up, there will be in the future tokens/assets standards such as BRC20/RGB/Taro that will make tokens available on bitcoin and these can be transferred to and used in lightning. So in the future we can have 3 layers, a token that depends on l2 that depends on a platform coin 😅

Reply:
> since L2/lightning is added to ctx similar to how tokens are added and depends heavily on the onchain coin

Yeah, this is what I was asking if I am correct or not. Thanks :)

> `warn!("Layer 2 coin {} can't be passive", self.ticker());`

It's not only used for making coins passive, that's why I wanted to write directly that function is ineffective